 Network rinha-nginx-2024q1  Creating
 Network rinha-nginx-2024q1  Created
 Container db  Creating
 Container db  Created
 Container oneabrante-api02-1  Creating
 Container oneabrante-api01-1  Creating
 Container oneabrante-api01-1  Created
 Container oneabrante-api02-1  Created
 Container oneabrante-nginx-1  Creating
 Container oneabrante-nginx-1  Created
Attaching to db, api01-1, api02-1, nginx-1
db       | 2024-02-28 04:07:40+00:00 [Note] [Entrypoint]: Entrypoint script for MariaDB Server 1:10.11.7+maria~ubu2204 started.
db       | 2024-02-28 04:07:40+00:00 [Warn] [Entrypoint]: /sys/fs/cgroup///memory.pressure not writable, functionality unavailable to MariaDB
db       | 2024-02-28 04:07:40+00:00 [Note] [Entrypoint]: Switching to dedicated user 'mysql'
db       | 2024-02-28 04:07:40+00:00 [Note] [Entrypoint]: Entrypoint script for MariaDB Server 1:10.11.7+maria~ubu2204 started.
db       | 2024-02-28 04:07:40+00:00 [Note] [Entrypoint]: Initializing database files
db       | 2024-02-28  4:07:40 0 [Warning] mariadbd: io_uring_queue_init() failed with errno 1
db       | 2024-02-28  4:07:40 0 [Warning] InnoDB: liburing disabled: falling back to innodb_use_native_aio=OFF
nginx-1  | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx-1  | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
nginx-1  | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
nginx-1  | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
nginx-1  | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
nginx-1  | /docker-entrypoint.sh: Configuration complete; ready for start up
api02-1  | Killed
api02-1  | Killed
api02-1  | Running migrations...
api01-1  | Killed
api01-1  | Killed
api01-1  | Running migrations...
api02-1  | Killed
api01-1  | Killed
api02-1  | Killed
api01-1  | Killed
nginx-1  | 2024/02/28 04:07:44 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.29.0.4:3000/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:07:44 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.29.0.3:3000/clientes/1/extrato", host: "localhost:9999"
api02-1  | Killed
api01-1  | Killed
api01-1  | Killed
api02-1  | Killed
api02-1  | 
api02-1  | > rinha_backend_2024@1.0.0 dev
api02-1  | > nodemon src/index.js
api02-1  | 
api01-1  | 
api01-1  | > rinha_backend_2024@1.0.0 dev
api01-1  | > nodemon src/index.js
api01-1  | 
api02-1  | [33m[nodemon] 3.1.0[39m
api02-1  | [33m[nodemon] to restart at any time, enter `rs`[39m
api02-1  | [33m[nodemon] watching path(s): *.*[39m
api02-1  | [33m[nodemon] watching extensions: js,mjs,cjs,json[39m
api02-1  | [32m[nodemon] starting `node src/index.js`[39m
api01-1  | [33m[nodemon] 3.1.0[39m
api01-1  | [33m[nodemon] to restart at any time, enter `rs`[39m
api01-1  | [33m[nodemon] watching path(s): *.*[39m
api01-1  | [33m[nodemon] watching extensions: js,mjs,cjs,json[39m
api01-1  | [32m[nodemon] starting `node src/index.js`[39m
api02-1  | Killed
api01-1  | Killed
nginx-1  | 2024/02/28 04:07:46 [error] 29#29: *4 no live upstreams while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api01-1 exited with code 137
api02-1 exited with code 137
db       | 
db       | 
db       | PLEASE REMEMBER TO SET A PASSWORD FOR THE MariaDB root USER !
db       | To do so, start the server, then issue the following command:
db       | 
db       | '/usr/bin/mariadb-secure-installation'
db       | 
db       | which will also give you the option of removing the test
db       | databases and anonymous user created by default.  This is
db       | strongly recommended for production servers.
db       | 
db       | See the MariaDB Knowledgebase at https://mariadb.com/kb
db       | 
db       | Please report any problems at https://mariadb.org/jira
db       | 
db       | The latest information about MariaDB is available at https://mariadb.org/.
db       | 
db       | Consider joining MariaDB's strong and vibrant community:
db       | https://mariadb.org/get-involved/
db       | 
db       | 2024-02-28 04:07:47+00:00 [Note] [Entrypoint]: Database files initialized
db       | 2024-02-28 04:07:47+00:00 [Note] [Entrypoint]: Starting temporary server
db       | 2024-02-28 04:07:47+00:00 [Note] [Entrypoint]: Waiting for server startup
db       | 2024-02-28  4:07:47 0 [Note] Starting MariaDB 10.11.7-MariaDB-1:10.11.7+maria~ubu2204 source revision 87e13722a95af5d9378d990caf48cb6874439347 as process 102
db       | 2024-02-28  4:07:47 0 [Note] InnoDB: Compressed tables use zlib 1.2.11
db       | 2024-02-28  4:07:47 0 [Note] InnoDB: Using transactional memory
db       | 2024-02-28  4:07:47 0 [Note] InnoDB: Number of transaction pools: 1
db       | 2024-02-28  4:07:47 0 [Note] InnoDB: Using crc32 + pclmulqdq instructions
db       | 2024-02-28  4:07:47 0 [Note] mariadbd: O_TMPFILE is not supported on /tmp (disabling future attempts)
db       | 2024-02-28  4:07:47 0 [Warning] mariadbd: io_uring_queue_init() failed with errno 1
db       | 2024-02-28  4:07:47 0 [Warning] InnoDB: liburing disabled: falling back to innodb_use_native_aio=OFF
db       | 2024-02-28  4:07:47 0 [Note] InnoDB: Initializing buffer pool, total size = 128.000MiB, chunk size = 2.000MiB
db       | 2024-02-28  4:07:47 0 [Note] InnoDB: Completed initialization of buffer pool
db       | 2024-02-28  4:07:47 0 [Note] InnoDB: File system buffers for log disabled (block size=4096 bytes)
db       | 2024-02-28  4:07:47 0 [Note] InnoDB: End of log at LSN=46862
db       | 2024-02-28  4:07:47 0 [Note] InnoDB: 128 rollback segments are active.
db       | 2024-02-28  4:07:47 0 [Note] InnoDB: Setting file './ibtmp1' size to 12.000MiB. Physically writing the file full; Please wait ...
db       | 2024-02-28  4:07:47 0 [Note] InnoDB: File './ibtmp1' size is now 12.000MiB.
db       | 2024-02-28  4:07:47 0 [Note] InnoDB: log sequence number 46862; transaction id 14
db       | 2024-02-28  4:07:47 0 [Note] Plugin 'FEEDBACK' is disabled.
db       | 2024-02-28  4:07:47 0 [Warning] 'user' entry 'root@5a5c69eefa16' ignored in --skip-name-resolve mode.
db       | 2024-02-28  4:07:47 0 [Warning] 'proxies_priv' entry '@% root@5a5c69eefa16' ignored in --skip-name-resolve mode.
db       | 2024-02-28  4:07:47 0 [Note] mariadbd: ready for connections.
db       | Version: '10.11.7-MariaDB-1:10.11.7+maria~ubu2204'  socket: '/run/mysqld/mysqld.sock'  port: 0  mariadb.org binary distribution
nginx-1  | 2024/02/28 04:07:48 [error] 29#29: *5 no live upstreams while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
db       | 2024-02-28 04:07:48+00:00 [Note] [Entrypoint]: Temporary server started.
db       | 2024-02-28 04:07:49+00:00 [Note] [Entrypoint]: Creating database rinhadb
db       | 2024-02-28 04:07:49+00:00 [Note] [Entrypoint]: Securing system users (equivalent to running mysql_secure_installation)
db       | 
db       | 2024-02-28 04:07:49+00:00 [Note] [Entrypoint]: /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/create_tables.sql
db       | 
db       | 
db       | 2024-02-28 04:07:49+00:00 [Note] [Entrypoint]: Stopping temporary server
db       | 2024-02-28  4:07:49 0 [Note] mariadbd (initiated by: unknown): Normal shutdown
db       | 2024-02-28  4:07:49 0 [Note] InnoDB: FTS optimize thread exiting.
db       | 2024-02-28  4:07:49 0 [Note] InnoDB: Starting shutdown...
db       | 2024-02-28  4:07:49 0 [Note] InnoDB: Dumping buffer pool(s) to /var/lib/mysql/ib_buffer_pool
db       | 2024-02-28  4:07:49 0 [Note] InnoDB: Buffer pool(s) dump completed at 240228  4:07:49
db       | 2024-02-28  4:07:49 0 [Note] InnoDB: Removed temporary tablespace data file: "./ibtmp1"
db       | 2024-02-28  4:07:49 0 [Note] InnoDB: Shutdown completed; log sequence number 59588; transaction id 33
db       | 2024-02-28  4:07:49 0 [Note] mariadbd: Shutdown complete
db       | 
db       | 2024-02-28 04:07:49+00:00 [Note] [Entrypoint]: Temporary server stopped
db       | 
db       | 2024-02-28 04:07:49+00:00 [Note] [Entrypoint]: MariaDB init process done. Ready for start up.
db       | 
db       | 2024-02-28  4:07:49 0 [Note] Starting MariaDB 10.11.7-MariaDB-1:10.11.7+maria~ubu2204 source revision 87e13722a95af5d9378d990caf48cb6874439347 as process 1
db       | 2024-02-28  4:07:49 0 [Note] InnoDB: Compressed tables use zlib 1.2.11
db       | 2024-02-28  4:07:49 0 [Note] InnoDB: Using transactional memory
db       | 2024-02-28  4:07:49 0 [Note] InnoDB: Number of transaction pools: 1
db       | 2024-02-28  4:07:49 0 [Note] InnoDB: Using crc32 + pclmulqdq instructions
db       | 2024-02-28  4:07:49 0 [Note] mariadbd: O_TMPFILE is not supported on /tmp (disabling future attempts)
db       | 2024-02-28  4:07:49 0 [Warning] mariadbd: io_uring_queue_init() failed with errno 1
db       | 2024-02-28  4:07:49 0 [Warning] InnoDB: liburing disabled: falling back to innodb_use_native_aio=OFF
db       | 2024-02-28  4:07:49 0 [Note] InnoDB: Initializing buffer pool, total size = 128.000MiB, chunk size = 2.000MiB
db       | 2024-02-28  4:07:49 0 [Note] InnoDB: Completed initialization of buffer pool
db       | 2024-02-28  4:07:49 0 [Note] InnoDB: File system buffers for log disabled (block size=4096 bytes)
db       | 2024-02-28  4:07:49 0 [Note] InnoDB: End of log at LSN=59588
db       | 2024-02-28  4:07:49 0 [Note] InnoDB: 128 rollback segments are active.
db       | 2024-02-28  4:07:49 0 [Note] InnoDB: Setting file './ibtmp1' size to 12.000MiB. Physically writing the file full; Please wait ...
db       | 2024-02-28  4:07:49 0 [Note] InnoDB: File './ibtmp1' size is now 12.000MiB.
db       | 2024-02-28  4:07:49 0 [Note] InnoDB: log sequence number 59588; transaction id 34
db       | 2024-02-28  4:07:49 0 [Note] InnoDB: Loading buffer pool(s) from /var/lib/mysql/ib_buffer_pool
db       | 2024-02-28  4:07:49 0 [Note] Plugin 'FEEDBACK' is disabled.
db       | 2024-02-28  4:07:49 0 [Warning] You need to use --log-bin to make --expire-logs-days or --binlog-expire-logs-seconds work.
db       | 2024-02-28  4:07:49 0 [Note] InnoDB: Buffer pool(s) load completed at 240228  4:07:49
db       | 2024-02-28  4:07:49 0 [Note] Server socket created on IP: '0.0.0.0'.
db       | 2024-02-28  4:07:49 0 [Note] Server socket created on IP: '::'.
db       | 2024-02-28  4:07:49 0 [Note] mariadbd: ready for connections.
db       | Version: '10.11.7-MariaDB-1:10.11.7+maria~ubu2204'  socket: '/run/mysqld/mysqld.sock'  port: 3306  mariadb.org binary distribution
nginx-1  | 2024/02/28 04:07:50 [error] 29#29: *6 no live upstreams while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:07:52 [error] 29#29: *7 no live upstreams while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:07:54 [error] 29#29: *8 no live upstreams while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:08:56 [error] 29#29: *9 upstream timed out (110: Connection timed out) while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.29.0.3:3000/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:09:14 [error] 29#29: *9 connect() failed (113: No route to host) while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.29.0.4:3000/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:09:19 [error] 29#29: *12 connect() failed (113: No route to host) while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.29.0.3:3000/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:09:19 [error] 29#29: *12 no live upstreams while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:09:21 [error] 29#29: *14 no live upstreams while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:09:23 [error] 29#29: *15 no live upstreams while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:09:28 [error] 29#29: *16 connect() failed (113: No route to host) while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.29.0.4:3000/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:09:28 [error] 29#29: *16 no live upstreams while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:09:33 [error] 29#29: *18 connect() failed (113: No route to host) while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.29.0.3:3000/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:09:33 [error] 29#29: *18 no live upstreams while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:09:35 [error] 29#29: *20 no live upstreams while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:09:37 [error] 29#29: *21 no live upstreams while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:09:42 [error] 29#29: *22 connect() failed (113: No route to host) while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.29.0.4:3000/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:09:42 [error] 29#29: *22 no live upstreams while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:09:47 [error] 29#29: *24 connect() failed (113: No route to host) while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.29.0.3:3000/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:09:47 [error] 29#29: *24 no live upstreams while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:09:49 [error] 29#29: *26 no live upstreams while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:09:51 [error] 29#29: *27 no live upstreams while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:09:57 [error] 29#29: *28 connect() failed (113: No route to host) while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.29.0.4:3000/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:09:57 [error] 29#29: *28 no live upstreams while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:10:02 [error] 29#29: *30 connect() failed (113: No route to host) while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.29.0.3:3000/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/28 04:10:02 [error] 29#29: *30 no live upstreams while connecting to upstream, client: 172.29.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1 exited with code 0
nginx-1 exited with code 0
db       | 2024-02-28  4:10:04 0 [Note] mariadbd (initiated by: unknown): Normal shutdown
db       | 2024-02-28  4:10:04 0 [Note] InnoDB: FTS optimize thread exiting.
db       | 2024-02-28  4:10:04 0 [Note] InnoDB: Starting shutdown...
db       | 2024-02-28  4:10:04 0 [Note] InnoDB: Dumping buffer pool(s) to /var/lib/mysql/ib_buffer_pool
db       | 2024-02-28  4:10:04 0 [Note] InnoDB: Buffer pool(s) dump completed at 240228  4:10:04
db       | 2024-02-28  4:10:05 0 [Note] InnoDB: Removed temporary tablespace data file: "./ibtmp1"
db       | 2024-02-28  4:10:05 0 [Note] InnoDB: Shutdown completed; log sequence number 61228; transaction id 35
db       | 2024-02-28  4:10:05 0 [Note] mariadbd: Shutdown complete
db       | 
db exited with code 0
